sales/send_quotations/invoice_and_deliver_to_different_address.rst

```diff
@@ -0,0 +1,48 @@
+===========================================
+Deliver and invoice to different addresses
+===========================================
+
+In Odoo you can configure different addresses for delivery and
+invoicing. This is key, not everyone will have the same delivery
+location as their invoice location.
+
+Activate the feature
+=====================
+
+Go to :menuselection:\`SALES --> Configuration → Settings\` and activate
```

Comment:
Don't escape the markdowns: The visual result is Go to :menuselection:`SALES --> Configuration → Settings` and activate the *Customer Addresses* feature.

Reply:
Fixed